SingularityUI/app/models/TaskResourceUsage.coffee

```diff
@@ -6,4 +6,18 @@ class TaskResourceUsage extends Model
 
     initialize: ({ @taskId }) =>
 
-module.exports = TaskResourceUsage
+    # Calculate CPU Usage by comparing previous usage to current usage
+    setCpuUsage: ->
+        previous = @get('previousUsage')
+        currentTime = @get('cpusSystemTimeSecs') + @get('cpusUserTimeSecs')
+        previousTime = previous.cpusSystemTimeSecs + previous.cpusUserTimeSecs
+        timestampDiff = @get('timestamp') - previous.timestamp
+        cpus_used = (currentTime - previousTime) / timestampDiff
+        cpuUsageExceeding = (cpus_used / @get('cpusLimit')) > 1.10
+        
+        if cpuUsageExceeding
```

Comment:
Would be useful to have another state between the "green" and red/danger exceeding states... like a yellow "warning" state when  `cpus_used` is >= 80% of the `cpusLimit`?